lib/plugins/create/templates/aws-alexa-typescript/serverless.yml

```diff
@@ -1,5 +1,7 @@
 service:
   name: aws-alexa-typescript
+#app: your-app-name
```

Comment:
I guess it's about credentials to Enterprise account. Wouldn't it be better to accompany it with comment as e.g.:

```yaml
# A credentials to eventual Serverless Enterprise account
# see https://serverless.com/acceleration/ for more details
```

Otherwise it gives no hint what those settings are about

Reply:
it's not a credential, but yeah, i'll check with product what copy they'd like

Reply:
Pretty sure it is, but is this only required for our AWS templates right now?

Reply:
Correct. SFE only supports AWS.